### DISCUSSION TITLE
PcBasic

### DISCUSSION THREAD

**Discussion by gferluga** [ish-app/ish#2276] [General]
I'm trying to execute PcBasic but I'm getting an error about py-xdg, even if it looks installed.

localhost:/opt/pcbasic# pcbasic
Traceback (most recent call last):
  File "/usr/local/bin/pcbasic", line 2, in <module>
    import pcbasic
  File "build/pcbasic/__init__.py", line 1, in <module>
  File "build/pcbasic/__main__.py", line 21, in <module>
  File "build/pcbasic/plat.py", line 63, in <module>
ImportError: No module named xdg.BaseDirectory

**Comment by saagarjha** [comment]
How did you install it?

**Reply by gferluga** [reply]
I simply extracted pcbasic-v1.2.15 and executed install.sh

Creating icon ... 
Creating uninstaller ... 

Checking dependencies ... 
checking Python module xdg ... NOT INSTALLED
checking Python module pygame ... NOT INSTALLED
checking Python module numpy ... NOT INSTALLED
checking Python module serial ... NOT INSTALLED
checking Python module parallel ... NOT INSTALLED
checking Python module pexpect ... NOT INSTALLED

WARNING: The following Python modules were not found:  xdg pygame numpy serial parallel pexpect
Please install them separately to ensure all PC-BASIC functionality works correctly.

How do I install those modules? Thx
